tests/Unit/Evolution/DiscontinuousGalerkin/SlopeLimiters/Test_WenoGridHelpers.cpp

```diff
@@ -0,0 +1,88 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <array>
+#include <cstddef>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Domain/Direction.hpp"
+#include "Domain/LogicalCoordinates.hpp"
+#include "Domain/Mesh.hpp"
+#include "Domain/Side.hpp"
+#include "Evolution/DiscontinuousGalerkin/SlopeLimiters/WenoGridHelpers.hpp"
+#include "NumericalAlgorithms/Spectral/Spectral.hpp"
+#include "Utilities/Gsl.hpp"
+
+namespace {
+
+template <size_t VolumeDim>
+void check_permutations(const Mesh<VolumeDim>& mesh) noexcept {
+  const auto check = [&mesh](
+      const std::array<DataVector, VolumeDim>& transformed_coords,
+      const size_t dim, const double offset) noexcept {
+    const DataVector expected_coords =
+        get<0>(logical_coordinates(mesh.slice_through(dim))) + offset;
```

Comment:
Is there an assumption here that allows you to get `<0>` in all cases?

Reply:
`mesh.slice_through(dim)` returns a 1D mesh